keg_elements/templates/keg_elements/forms/horizontal_b4.html

```diff
@@ -127,9 +127,10 @@
  #}
 {% macro checkbox_field(field) -%}
     {% call div_form_group(field, **kwargs) %}
-        <div class="unlabeled-group checkbox">
-            <label>
-                {{ field(type='checkbox', **kwargs) }} {{ field.label }}
+        <div class="unlabeled-group checkbox form-check">
+            {{ field(type='checkbox', class_='form-check-input is-invalid' if field.errors else 'form-check-input', **kwargs) }}
```

Comment:
The behavior here is `kwargs` will override the default `class_` here if it is set on the `kwargs`, correct? I think that is what we want, but just want to make sure that will happen.